azure-webapp-maven-plugin/src/main/java/com/microsoft/azure/maven/webapp/serializer/V1ConfigurationSerializer.java

```diff
@@ -0,0 +1,98 @@
+        linuxRuntimeStackMap.put(RuntimeStack.JAVA_8_JRE8, "jre8");
+    }
+
+    @Override
+    public void saveToXML(WebAppConfiguration webAppConfiguration, Element configurationElement)
+        throws MojoFailureException {
+        createOrUpdateAttribute("schemaVersion", "V1", configurationElement);
+        createOrUpdateAttribute("resourceGroup", webAppConfiguration.getResourceGroup(), configurationElement);
+        createOrUpdateAttribute("appName", webAppConfiguration.getAppName(), configurationElement);
+        createOrUpdateAttribute("region", webAppConfiguration.getRegion().name(), configurationElement);
+        createOrUpdateAttribute("pricingTier",
+            PricingTierEnum.getPricingTierStringByPricingTierObject(webAppConfiguration.getPricingTier()),
+            configurationElement);
+
+        coinfigRuntime(webAppConfiguration, configurationElement);
+
+        XMLUtils.removeNode(configurationElement, "deploymentSlot");
+        if (webAppConfiguration.getDeploymentSlotSetting() != null) {
+            configurationElement.add(createDeploymentSlotNode(webAppConfiguration.getDeploymentSlotSetting()));
+        }
+    }
+
+    private void coinfigRuntime(WebAppConfiguration webAppConfiguration, Element domElement)
+        throws MojoFailureException {
+        // remove old runtime configs
+        removeRuntimeConfigs(domElement);
+        switch (webAppConfiguration.getOs()) {
+            case Linux:
+                configLinuxRunTime(webAppConfiguration, domElement);
+                break;
+            case Windows:
+                configWindowsRunTime(webAppConfiguration, domElement);
+                break;
+            case Docker:
+                configDockerRunTime(webAppConfiguration, domElement);
+                break;
+            default:
+                throw new MojoFailureException("The value of <os> is unknown.");
+        }
+    }
+
+    private void removeRuntimeConfigs(Element domElement) {
+        XMLUtils.removeNode(domElement, "javaVersion");
+        XMLUtils.removeNode(domElement, "webContainer");
+        XMLUtils.removeNode(domElement, "linuxRuntime");
+        XMLUtils.removeNode(domElement, "image");
+        XMLUtils.removeNode(domElement, "serverId");
+        XMLUtils.removeNode(domElement, "registryUrl");
+    }
+
+    private void configWindowsRunTime(WebAppConfiguration webAppConfiguration, Element domElement) {
+        domElement.add(XMLUtils.createSimpleElement("javaVersion",
+            webAppConfiguration.getJavaVersion().toString()));
+        domElement.add(XMLUtils.createSimpleElement("webContainer",
+            webAppConfiguration.getWebContainer().toString()));
+    }
+
+    private void configLinuxRunTime(WebAppConfiguration webAppConfiguration, Element domElement) {
+        domElement.add(XMLUtils.createSimpleElement("linuxRuntime",
+            linuxRuntimeStackMap.get(webAppConfiguration.getRuntimeStack())));
```

Comment:
Can we use the result of `RuntimeStack::toString()`?

Reply:
No, RuntimeStack::toString() will return result with jre version which was not supported value for our configuration  (https://github.com/Microsoft/azure-maven-plugins/blob/develop/azure-webapp-maven-plugin/README.md#details-explanation-of-some-configurations)

Reply:
V1Configuration use RuntimeStack.toString() as linux runtime directly, so change it in next commit.